### PR TITLE
Ana/5143 display seeds on user menu

### LINKED ISSUE #5143: allow users to display their seeds on mobile
On the right of a user account list entry add three dots. This opens the slight right menu like in the extension.
For Ledger and Explore accounts it would show "Delete Account". For local accounts it shows "Show Seed" and "Delete Account". Clicking on those buttons opens a modal which has the same components as the flow in the extension.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #5143

**Description:**

image

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f901a4f3249ef32b97d48ca--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
New version in the works, applying Jordan's suggestions

File: app/src/components/account/AccountList.vue [hunk 1]
```diff
@@ -64,24 +71,37 @@ export default {
...
+      type: Boolean,
+      default: false,
     },
   },
   data: () => ({
     openAccount: undefined,
     isExtension: config.isExtension,
   }),
+  computed: {
+    ...mapGetters([`networks`]),
```
Comment (faboweb) [inline app/src/components/account/AccountList.vue:90]: where is this needed?

File: app/src/components/account/UserMenu.vue [hunk 2]
```diff
@@ -72,6 +66,15 @@
           <span>Add an address</span>
           <i class="material-icons">add_circle</i>
         </div>
+        <div
+          id="show-seed"
```
Comment (faboweb) [inline app/src/components/account/UserMenu.vue:70]: wrong?

File: app/src/components/account/UserMenu.vue [hunk 3]
```diff
@@ -182,6 +185,7 @@ export default {
         let allAddressesWithAddressRole = await this.getAllAddressesRoles(
           allAddresses.filter(({ address }) => address)
         )
+        this.$store.commit(`setUserAccounts`, allAddressesWithAddressRole)
```
Comment (faboweb) [inline app/src/components/account/UserMenu.vue:188]: 🙈

File: app/src/routes.js [hunk 4]
```diff
@@ -270,7 +270,7 @@ export default (store) => {
       component: require("account/RevealSeedModal").default,
     },
     {
-      path: `/delete/:networkId/:address`,
+      path: `/delete/:address/:addressNetworkId`,
```
Comment (faboweb) [inline app/src/routes.js:273]: why the change?

File: app/tests/unit/specs/components/account/AccountMenu.spec.js [hunk 5]
```diff
@@ -7,14 +7,17 @@ describe(`AccountMenu`, () => {
...
+          networkId: `cosmos-hub-mainnet`,
+          sessionType: `local`,
+        },
       },
       stubs: [`router-link`],
     })
   })
 
-  it(`should show the AccountMenu page`, async () => {
+  it.skip(`should show the AccountMenu page`, async () => {
```
Comment (faboweb) [inline app/tests/unit/specs/components/account/AccountMenu.spec.js:20]: ?

File: app/src/routes.js (hunk 4 above)
Comment (Bitcoinera) [reply app/src/routes.js:273]: networkId was being problematic in the `setNetwork` function since it is the same name for the network param. The order doesn't actually matter. I can restore the previous order

File: app/src/components/account/UserMenu.vue (hunk 2 above)
Comment (Bitcoinera) [reply app/src/components/account/UserMenu.vue:70]: true, forgot to change it

File: app/tests/unit/specs/components/account/AccountMenu.spec.js (hunk 5 above)
Comment (Bitcoinera) [reply app/tests/unit/specs/components/account/AccountMenu.spec.js:20]: yeah, it fails. I don't understand why. Please help

File: app/src/components/account/UserMenu.vue (hunk 3 above)
Comment (Bitcoinera) [reply app/src/components/account/UserMenu.vue:188]: :shrug:  how to do it otherwise?

Comment (Bitcoinera) [reply app/src/components/account/UserMenu.vue:188]: You want to recalculate all them in ManageAccounts all over?

Comment (faboweb) [reply app/src/components/account/UserMenu.vue:188]: it's a fine hack as we will move into lunie3. in general you should move data logic into the store if it is used in several places. so we would likely call `getAllAccounts` which would in the store get all the accounts and deliver them to the component.

File: app/src/routes.js (hunk 4 above)
Comment (faboweb) [reply app/src/routes.js:273]: alright, and why the reordering?

File: app/tests/unit/specs/components/account/AccountMenu.spec.js (hunk 5 above)
Comment (faboweb) [reply app/tests/unit/specs/components/account/AccountMenu.spec.js:20]: next time, tell me and not just leave it in ;)

File: app/src/routes.js (hunk 4 above)
Comment (faboweb) [reply app/src/routes.js:273]: leave it, but always better to minimize changes to not create confusion

File: app/tests/unit/specs/components/account/AccountMenu.spec.js [hunk 6]
```diff
@@ -17,7 +17,7 @@ describe(`AccountMenu`, () => {
     })
   })
 
-  it.skip(`should show the AccountMenu page`, async () => {
+  it(`should show the AccountMenu page`, async () => {
```
Comment (Bitcoinera) [inline app/tests/unit/specs/components/account/AccountMenu.spec.js]: Thank you :pray:

**Comment by faboweb** [comment]
ups merged too quick, can you create a PR with the last missing changes from the comments?

**Comment by Bitcoinera** [comment]
okii

File: app/tests/unit/specs/components/account/AccountMenu.spec.js (hunk 5 above)
Comment (Bitcoinera) [reply app/tests/unit/specs/components/account/AccountMenu.spec.js]: yeah, my intention was telling, not hiding it :sweat_smile: I just had to leave at the time

File: app/src/components/account/AccountList.vue (hunk 1 above)
Comment (Bitcoinera) [reply app/src/components/account/AccountList.vue:90]: removed
